modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -105,3 +105,14 @@
 .sidebar {
 	transition: transform 0.5s ease;
 }
+
+.text-truncate-reverse {
+	direction: rtl;
+}
+
+.loading-animation {
```

Comment:
Why do we need to redefine `loading` styles?

Reply:
I just view that you wrapped the `<ClayLoadingIndicator />` in Content Dashboard, I think that should avoid to add this custom CSS and center it in the sidebar panel. I'm going to try it 😄 

```js
<div className="align-items-center d-flex loading-indicator-wrapper">
	<ClayLoadingIndicator small />
</div>
```

Reply:
I'm having problems because the markup is different in Content Performance than in Content Dashboard > Content Performance. The second one is not taking the full height of the sidebar (limited by the content, inside `Sidebar.Body`) and `align-items-center d-flex` it's not working. So I think I'm going to remove this custom CSS and be happy 😂